src/features/organizations/components/OrganizationTree.tsx

```diff
@@ -53,6 +54,10 @@ function OrganizationTree({
   orgId,
 }: OrganizationTreeProps): JSX.Element {
   const theme = useTheme();
+  const sortedTreeItems = useMemo(() => {
+    const sorted = treeItemData.sort((a, b) => a.title.localeCompare(b.title));
+    return sorted;
+  }, [treeItemData.length]);
```

Comment:
Correct me if I'm wrong, but I believe this only sorts the top-level, which is not enough. We also want to sort the children of each organization, and all of the children of those children, etc.

Reply:
Aaaah yeah you are right. I will add more logic to it!

Reply:
I believe the best solution would be to handle the sorting in the rendering of each "level", rather than here in the top-level.